go/ql/test/library-tests/semmle/go/frameworks/Beego/test.go

```diff
@@ -22,6 +28,27 @@ type bindMe struct {
 	c subBindMe
 }
 
+// BAD: using user-provided data as paths in file-system operations
+func fsOpsV2Test(ctx *Beegov2Context.Context, c *beegov2.Controller) {
+	input := ctx.Input
+	untrusted := input.Data()["someKey"].(string)
+	buffer := make([]byte, 10)
+	_ = c.SaveToFileWithBuffer("filenameExistsInForm", untrusted, buffer)
+	beegoOutput := Beegov2Context.BeegoOutput{}
+	beegoOutput.Download(untrusted, "license.txt")
+}
+
+// BAD: using user-provided data as paths in file-system operations
+func fsOpsTest(ctx *context.Context, c *beego.Controller, fs beego.FileSystem) {
```

Comment:
Why did you move this? The expected result changes would be easier to read if you had left it where it was and put the new function after it.